CUDADataFormats/TrackingRecHit/interface/TrackingRecHit2DHeterogeneous.h

```diff
@@ -101,11 +103,15 @@ TrackingRecHit2DHeterogeneous<Traits>::TrackingRecHit2DHeterogeneous(uint32_t nH
   m_store32 = Traits::template make_device_unique<float[]>(nHits * n32 + 11, stream);
```

Comment:
as suggested by @slava77 [here](https://github.com/cms-sw/cmssw/pull/31721/files#r521097079), could you clarify if `nHits * n32 + 11` should be `nHits * n32 + phase1PixelTopology::numberOfLayers + 1` ?
```suggestion
  m_store32 = Traits::template make_device_unique<float[]>(nHits * n32 + phase1PixelTopology::numberOfLayers + 1, stream);
```

Reply:
indeed.